Add `daily stand-ups` section to home page

cvita opened the pull request:
This PR embeds a Twitter timeline and a list of recent GitHub commits as a subcomponent for the site's home page.

cvita committed:
Add more descriptive site title

cvita committed:
Add `react-load-script` for Twitter widget embed

cvita committed:
Add `<StandUp />` which displays twitter timeline

Timeline displays daily Stand-ups.

cvita committed:
Add tests for `testimonial` and `commits`

Both are props managed by Redux.

cvita committed:
Add `gitHub` client methods for `commits` redux prop

cvita committed:
Add `fetchCommits` action, saga, reducer

For `<RecentCommits />` subcomponent in `<StabdUp />`.

cvita committed:
Add `commits` redux prop

cvita committed:
Add `<RecentCommits />` subcomponent to `<StandUp />`

cvita committed:
Add `<StandUp />` as subcomponent to `<Home />`

cvita committed:
Add lazy-load images

To help offset the new calls to Twitter and GitHub.

VitaC123 commented:
Looking for opportunities to optimize Lighthouse performance score before merging.

cvita committed:
Cut `react-lazy-load` to replace with `react-lazyload`

Improved performance, smoother feel while scrolling.

cvita committed:
Add `rel=preload` to Bootstrap and web-font

Bootstrap no longer a blocking stylesheet. Web-font relocated from `index.css`.

cvita committed:
Add `styleSheetLoaded` as new redux prop

Useful if all other props finish updating before Bootstrap is finished loading.

cvita committed:
Add event listener to call `refreshStyleSheetStatus()`

Also adds condition display above fold elements if Bootstrap is loaded.

cvita committed:
Fix minor style bug

Play button size and margin rules not affecting.

cvita committed:
Fix lazyload placeholder height for `<StandUp />`

cvita committed:
Add `rel=preload` for hero image

Further reduces first meaningful paint time.

cvita committed:
Refactor components

More consistent patterns as established in rest of app.

cvita committed:
Cut `react-load-script` add `react-twitter-wigets`

Faster and more relevant to project. Also simplifies custom styles.

cvita committed:
Add style `marginBottom` for consistency

VitaC123 commented:
**Ready to merge!**

Repeat testing on the staging site yields performance scores between 74 and 85. These are similar scores to the site before adding the daily standups via Twitter and recent commits via GitHub.

If needed later, I may tackle the remaining blocking stylesheet issue by putting essential styles inline, while the stylesheet loads async.

<img width="441" alt="screen shot 2017-11-14 at 3 18 19 pm" src="https://user-images.githubusercontent.com/17498763/32810265-1994528a-c94f-11e7-8d10-bbb1faad389c.png">
